mobile/src/home-view.js

```diff
@@ -263,25 +276,25 @@ const s = ReactNative.StyleSheet.create({
     justifyContent: 'center',
   },
   buttonText: {
-    fontSize: 20,
+    fontSize: 18,
     color: '#fff',
     textAlign: 'center',
   },
   welcomeBox: {
-    backgroundColor: new Color(client.primaryColor).limitLightness(0.2).rgbString(),
-    margin: 10,
-    borderRadius: 5,
+    backgroundColor: "#FFFFFF",
+    marginVertical: 10,
   },
   welcomeTitle: {
-    color: '#fff',
+    color:'#364247',
     fontSize: 24,
     textAlign: 'center',
     marginTop: 20,
   },
   welcomeText: {
-    color: '#fff',
+    color: '#364247',
```

Comment:
Since you are using this color 4 times, use something like `const charcoal = '#364247'`